assembl/static2/tests/unit/components/common/scrollOnePageButton.spec.jsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
 
-import ScrollOnePageButton from '../../../../js/app/components/common/scrollOnePageButton';
+import { ScrollOnePageButton } from '../../../../js/app/components/common/scrollOnePageButton';
 
 describe('ScrollOnePageButton component', () => {
   it('should render a button to scroll one page down', () => {
-    const component = renderer.create(<ScrollOnePageButton hidden={false} />);
+    const component = renderer.create(<ScrollOnePageButton screenHeight={768} hidden={false} />);
```

Comment:
If they aren't avaiable yet, you can put the screen dimensions in the constans.js file and refer to them here. DRY.

Reply:
I don't think I should add these to constants file since it's the screen dimensions in the tests only. I add a testConstants file ?